components/TestimonialLists/index.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <div>
+    <Card :testimonials="testimonials" />
+  </div>
+</template>
+
+<script>
+
+export default {
+
+  data () {
+    return {
+      testimonials: []
+    }
+  },
+  async fetch () {
+    const response = await this.$axios.get('/testimonials')
+    this.testimonials = response.data.data
+  }
+}
+</script>
+
+<style>
```

Comment:
minor: 

 Why did you write this code when it was not used? I think you can remove it for better.